a3.c

```diff
@@ -184,7 +184,7 @@ void marian_a3_proc_status(struct marian_card* marian, struct snd_info_buffer* b
 
 	marian_proc_status_generic(marian, buffer);
 
-	buf = marian->dmabuf.area;
+	buf = (uint32_t *) marian->dmabuf.area;
```

Comment:
Not a real kernel type, please use u32 instead.

You can fix this up later, as that needs to be done for all types in the driver.